### PR TITLE
Fix #211: Implement persistent session.

### LINKED ISSUE #211: Persistent sessions ?
Use cases

* Alice sends a link to an object to Bob. Since Bob had authenticated him with the Kinto-admin a few hours earlier, he can open the page on the object directly
* During development, I refresh the page. I have to login again and go back to the page I was hacking

### PR DESCRIPTION
**Pull request by n1k0**
Refs #211. Deployed to gh-pages.

Couldn't use `sessionStorage` as persisted data only lives for a single given opened page. Using `localStorage` works fine for this use case, and is persistent across browser restarts. Clearing the session is just matter of logging out.

r=? @Natim @glasserc @leplatrem

### CODE REVIEW

File: plugins/signoff/index.js [hunk 1]
```diff
@@ -16,8 +16,18 @@ function requestSignoff() {
...
+  const {groups} = bucket;
+  const {serverInfo: {user: {id}}} = session;
+
+  const editorGroup = groups.find(group => group.id === "editors");
+  const isEditor = editorGroup && editorGroup.members.includes(id);
+
+  if (!isEditor) {
+    yield put(notifyError("You are not an editor!", new Error("XXX")));
+    return;
+  }
```
Comment (n1k0) [inline plugins/signoff/index.js]: Leftovers from another branch, please disregard.

File: package.json [hunk 2]
```diff
@@ -76,6 +76,7 @@
     "gh-pages": "^0.11.0",
     "html": "0.0.10",
     "iflow-react-router": "^1.1.17",
+    "ignore-styles": "^5.0.1",
```
Comment (n1k0) [inline package.json]: This is required for running code in node where css modules are required (we now import symbols from `app.js`)

**Comment by n1k0** [comment]
This is now ready for final review. r=? @Natim

**Comment by n1k0** [comment]
Ah, @Natim and @leplatrem are off and @glasserc should be sleeping. I guess I'm gonna land this with a green build methinks.

**Comment by glasserc** [comment]
r+

**Pull request merged**